data/hlint.yaml

```diff
@@ -443,6 +443,8 @@
     - warn: {lhs: flip (=<<), rhs: (>>=)}
     - hint: {lhs: \x -> f x >>= g, rhs: f Control.Monad.>=> g}
     - hint: {lhs: \x -> f =<< g x, rhs: f Control.Monad.<=< g}
+    - hint: {lhs: (>>= f) . g, rhs: f Control.Monad.<=< g}
```

Comment:
```suggestion
    - hint: {lhs: (>>= g) . f, rhs: f Control.Monad.>=> g}
```
Keep the symmetry?

Reply:
I like my version slightly better since it preserves the relative order of `f` and `g`. Imagine the original code being `f1 . f2 . (>>= f3) . f4 . f5`